driver/test/java/io/crate/client/jdbc/integrationtests/CrateJDBCConnectionTest.java

```diff
@@ -85,7 +85,7 @@ public void testConnectionWithCustomSchema() throws Exception {
             assertThat(resultSet.getLong(1), is(1L));
 
             resultSet = statement.executeQuery(
-                "select collect_set(schema_name) from information_schema.tables where table_name = 't'");
+                "select collect_set(table_schema) from information_schema.tables where table_name = 't'");
```

Comment:
not compatible with `0.56`
you can wait for https://github.com/crate/crate-jdbc/pull/195